ee/tabby-schema/graphql/schema.graphql

```diff
@@ -885,6 +889,11 @@ type ThreadAssistantMessageCreated {
   id: ID!
 }
 
+type ThreadAssistantMessageReadingCode {
```

Comment:
@liangfung let's discuss this in weekly sync on how to present this info in frontend.